test/controllers/GuidanceControllerSpec.scala

```diff
@@ -26,7 +26,7 @@ import play.api.test.FakeRequest
 import play.api.test.Helpers._
```

Comment:
The test coverage for this class has dropped a little. We could do with a few additional tests to raise the coverage.